pyuvdata/uvdata.py

```diff
@@ -1550,19 +1596,67 @@ def __add__(self, other, run_check=True, check_extra=True,
         ValueError
```

Comment:
Though we should mention that a `ValueError` is also raised if `phase_center_radec` is not the right size.